.gitignore

```diff
@@ -29,6 +29,7 @@ test_vec
 
 # Build directory in source
 build/
+opm-parallel/
```

Comment:
Please remove and use the command
```
echo opm-parallel >> .git/info/exclude 
```
in your checkout to ignore this build directory locally.